main/modes/platformer/mode_platformer.c

```diff
@@ -1777,20 +1316,59 @@ void updateShowHighScores(platformer_t *self){
     )){
         self->menuState = 0;
         self->menuSelection = 0;
+        buzzer_stop();
         changeStateTitleScreen(self);
     }
 
     drawShowHighScores(self->disp, &(self->radiostars), self->menuState);
     drawPlatformerHighScores(self->disp, &(self->radiostars), &(self->highScores), &(self->gameData));
 
+    updateLedsShowHighScores(&(self->gameData));
+
     self->prevBtnState = self->btnState;
     self->gameData.prevBtnState = self->prevBtnState;
 }
 
 void drawShowHighScores(display_t *d, font_t *font, uint8_t menuState){
-    if(menuState == 3){
-        drawText(d, font, c555, "Your name registrated.", 24, 32);
+    if(platformer->easterEgg){
+        drawText(d, font, highScoreNewEntryColors[(platformer->gameData.frameCount >> 3) % 4], "Happy Birthday, Evelyn!", 20, 32);
+    } else if(menuState == 3){
+        drawText(d, font, redColors[(platformer->gameData.frameCount >> 3) % 4], "Your name registrated.", 24, 32);
```

Comment:
Registrated? Registered?

Reply:
The wording is directly lifted from the Mario Bros. Arcade game. Always thought it was hilarious. I’m keeping it.

Reply:
Thank you so much a for to playing my game